math/gnuplot/Portfile

```diff
@@ -15,7 +15,6 @@ categories          math science
 license             Restrictive
 # luckily nothing has to link against this
 installs_libs       no
-platforms           darwin
```

Comment:
What's the reason for this change?
Are we removing `platforms` from all ports now?
(The aquaterm variant is certainly incompatible with other platforms.)

The update itself looks ok.

Reply:
> What's the reason for this change? Are we removing `platforms` from all ports now? (The aquaterm variant is certainly incompatible with other platforms.)

since MacPorts v2.7 this is the default that is set by base, see https://github.com/macports/macports-base/commit/d9181903e6fda245fda3dab3bbe3dcceb2650323.